pyproject.toml

```diff
@@ -92,9 +91,14 @@ filterwarnings = [
     "ignore:.*divide by zero encountered in det.*:RuntimeWarning:",
     "ignore:.*invalid value encountered in det.*:RuntimeWarning:",
     "ignore:..*env.action_masks to get variables from other wrappers is deprecated and will be removed in v1.0.*:UserWarning:",
-
+    "ignore::SyntaxWarning:.*(docplex).*",
+    "ignore:.*pytorch.*:UserWarning:",
+    "ignore::DeprecationWarning:.*",
```

Comment:
```suggestion
```
As you stated already, this is a bit too generic. We should only exclude explicit warnings if possible.